docs/source/geo_def.rst

```diff
@@ -88,10 +89,83 @@ based on area extent and a proj4-string or a list of proj4 arguments.
  Number of rows: 425
  Area extent: (-5326849.0625, -5326849.0625, 5326849.0625, 5326849.0625)
 
+The function **from_params** attempts to find shape and area_extent with the given data below:
+
+Required (positional) arguments:
+
+* **description**: Description
+* **projection**: projection parameters as a proj4_dict or proj4_string
+
+Optional (keyword) arguments:
 
-The **load_area** function can be used to parse area definitions from a
-configuration file. Assuming the file **areas.yaml** exists with the following
-content
+* **area_id**, **proj_id**, and **area_extent**: same as AreaDefinition
+* **units**: Default projection units (meters/radians/degrees)
+* **shape**: (height, width). Note: if height = width, then only height or width needs to be passed.
```

Comment:
Maybe add `in number of pixels` to this description and remove the mention of shape in the `units` part below. Also maybe remove the Note part from each of these and put it after and say that `shape, pixel_size, and radius can be specified with one value when they are the same (I'm starting to think accepting one value was a mistake code-wise, but you've already done it so leave it).

I think you can combine the "where" descriptions with the individual arguments here and simplify the description to something like `x and y coordinate of upper left corner...`.